scio-elasticsearch/src/main/java/com/spotify/scio/elasticsearch/ElasticsearchIO.java

```diff
@@ -0,0 +1,231 @@
+package com.spotify.scio.elasticsearch;
```

Comment:
Missing license header here. (We should set up automated checks for license headers on Java sources too, Scalastyle obviously only checks Scala sources).